src/suite.jl

```diff
@@ -38,19 +38,33 @@ function verify_case(dir;verbose=false)
     try 
         fns = readdir(dir;join=true)
         model_fn = filter(endswith("l2v3.xml"), fns)[1]
+        case_no = basename(dirname(model_fn))
         settings = setup_settings_txt(filter(endswith("settings.txt"), fns)[1])
         results = CSV.read(filter(endswith("results.csv"), fns)[1], DataFrame)
+        
         ml = SBML.readSBML(model_fn, doc -> begin
-            set_level_and_version(3, 1)(doc)
+            set_level_and_version(3, 2)(doc)
             convert_simplify_math(doc)
             end)
+        rs = ReactionSystem(ml)
         sys = ODESystem(ml)
+        statenames = [string(s.f.name) for s in sys.states]
+        
         ts = LinRange(settings["start"], settings["duration"], settings["steps"])
         prob = ODEProblem(sys, Pair[], (settings["start"], Float64(settings["duration"])); saveat=ts)
-        sol = solve(prob, CVODE_BDF(); abstol=settings["absolute"], reltol=settings["relative"])
-        solm = Array(sol)'
-        m = Matrix(results[1:end-1, 2:end])
+        sol = solve(prob, CVODE_BDF(); abstol=settings["absolute"]/20, reltol=settings["relative"]/20)

```

Comment:
why divide by 20?

Reply:
sorry. i just tried sth. fixed in a2e2dce .